src/main/java/com/spotify/reaper/resources/CommonTools.java

```diff
@@ -252,6 +253,25 @@ public static RepairSchedule storeNewRepairSchedule(
                                    repairParallelism, intensity,
                                    DateTime.now());
     scheduleBuilder.owner(owner);
+    
+    Collection<RepairSchedule> repairSchedules = context.storage.getRepairSchedulesForClusterAndKeyspace(repairUnit.getClusterName(), repairUnit.getKeyspaceName());
+    for(RepairSchedule sched:repairSchedules){
+    	Optional<RepairUnit> repairUnitForSched = context.storage.getRepairUnit(sched.getRepairUnitId());
```

Comment:
blocking: replace tabs with whitespace. (happens in many of the edited files in this PR)